OSSM-4041: Update infrastructure node content (#60744)
openshift/openshift-docs: modules/ossm-config-control-plane-infrastructure-node-cli.adoc

// * service_mesh/v2x/ossm-deployment-models.adoc

:_content-type: PROCEDURE
[id="ossm-config-control-plane-infrastructure-node-cli_{context}"]
= Configuring all control plane components to run on infrastructure nodes with the CLI

> You can configure all of the {SMProductShortName} control plane components to run on infrastructure nodes using the CLI. This task should only be performed if all of the components deployed by the {SMProductShortName} control plane, including Istiod, Ingress Gateway, and Egress Gateway, along with optional elements, such as Prometheus, Grafana, and Distributed Tracing, are running on infrastructure nodes. 

Reviewer: I don't understand the second sentence. It says that this task (moving stuff to infra nodes) should only be performed if that same stuff *is running* on infra nodes. Should the sentence read as follows?
```suggestion
You can configure all of the {SMProductShortName} control plane components to run on infrastructure nodes using the CLI. This task should only be performed if you want all of the components deployed by the {SMProductShortName} control plane, including Istiod, Ingress Gateway, and Egress Gateway, along with optional elements, such as Prometheus, Grafana, and Distributed Tracing, to run on infrastructure nodes. 
```

> If the control plane runs on a worker node, skip this task.

Reviewer: Similar thing here. The purpose of this task is to move the control plane from worker nodes to infra nodes, so the sentence doesn't make sense to me.

Author: Done

// * service_mesh/v2x/ossm-deployment-models.adoc

:_content-type: PROCEDURE
[id="ossm-config-control-plane-infrastructure-node-cli_{context}"]

> = Configuring all control plane components to run on infrastructure nodes with the CLI

Author: done

Perform this task if all of the components deployed by the {SMProductShortName} control plane, including Istiod, Ingress Gateway, and Egress Gateway, along with optional elements, such as Prometheus, Grafana, and Distributed Tracing, will run on infrastructure nodes. 

> If the control plane will run on a worker node, skip this task.

Author: Prereqs have been added to all the tasks. Thanks for pointing this out.

// * service_mesh/v2x/ossm-deployment-models.adoc

:_content-type: PROCEDURE
[id="ossm-config-control-plane-infrastructure-node-cli_{context}"]
= Configuring all control plane components to run on infrastructure nodes using the CLI

> Perform this task if all of the components deployed by the {SMProductShortName} control plane, including Istiod, Ingress Gateway, and Egress Gateway, along with optional elements, such as Prometheus, Grafana, and Distributed Tracing, will run on infrastructure nodes. 

Reviewer 2: ```suggestion
Perform this task if all of the components deployed by the {SMProductShortName} control plane will run on infrastructure nodes. These components include Istiod, Ingress Gateway, and Egress Gateway along with optional components, such as Prometheus, Grafana, and Distributed Tracing,
```
- This sentence is too information dense. Consider breaking into 2 sentences, maybe as I suggest here?
- Prometheus and Grafana are `components`, and not `elements`, I would say.

Author: Done

Author: done

If the control plane will run on a worker node, skip this task.

.Prerequisites

* You have installed the {SMProductName} Operator.

> * You must have an account with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.

Reviewer 2: ```suggestion
* You are logged in as a user with the `cluster-admin` role. 
* If you use {product-dedicated}, you are logged in as a user with the `dedicated-admin` role.
```
Do these accounts just need to exist? Or does the user need to be logged in with one of these accounts? If the latter, edit per my suggestion.

Author: Done
